src/SamplesApp/Benchmarks.Shared/Suite/Windows_UI_Xaml/ResourceDictionaryBench/XamlControlsResourcesCreationRetrievalBenchmark.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using BenchmarkDotNet.Attributes;
+using Microsoft.UI.Xaml.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace SamplesApp.Benchmarks.Suite.Windows_UI_Xaml.ResourceDictionaryBench
+{
+	public class XamlControlsResourcesCreationRetrievalBenchmark
+	{
+		[Benchmark]
+		public void Create_XamlControlsResources_And_Retrieve_Style()
+		{
+			var xcr = new XamlControlsResources();
+
+			var style = xcr["ListViewItemExpanded"] as Style;
+			var templateSetter = style.Setters.OfType<Setter>().First(s => s.Property == Control.TemplateProperty);
+			var template = templateSetter.Value;
```

Comment:
Codacy found an issue: [Remove the unused local variable 'template'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6177554)